.github/workflows/main.yml

```
@@ -160,3 +160,9 @@ jobs:
           $env:TASK = "${{ matrix.task }}"
           conda init powershell
           & "$env:GITHUB_WORKSPACE/.ci/test_windows.ps1"
+  allgood:
+    runs-on: ubuntu-latest
+    needs: [test]
+    steps:
+    - name: echo allgood
+      run: echo "allgood"
```

I don't like `simple-task` as it's not very explanatory:

how about:

```suggestion
  all-successful:
    runs-on: ubuntu-latest
    needs: [test]
    steps:
    - name: "Proclaim that all jobs in the workflow are successful, for easy [requires]"
      run: echo "🎉"
```

---

echoing an emoji is ok with me! I think this test name is too long. Can you please just call is `check all tests succeeded`, and then add a comment with the discussion link like I recommended in https://github.com/microsoft/LightGBM/pull/3351#discussion_r483687906?

---

all-github-actions-tests isn't good because it's already in the "Github Actions" workflow namespace

check isn't quite accurate either, so I think commemorate is more correct:

```suggestion
  all-successful:  # https://github.community/t/is-it-possible-to-require-all-github-actions-tasks-to-pass-without-enumerating-them/117957/4?u=graingert
    runs-on: ubuntu-latest
    needs: [test]
    steps:
    - name: commemorate that all tests succeeded
      run: echo "🎉"
```

---

We have many maintainers and contributors on this project who are not native English speakers, so I have a strong preference for plain language, avoiding words like `proclaim` or `commemorate`.

Please call the step `check all tests succeeded` and please put the comment on its own line (not inline with `all-successful: `).